src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -107,7 +101,15 @@ export const ArtworkSidebarBidActionFragmentContainer = createFragmentContainer(
   ArtworkSidebarBidAction,
   graphql`
     fragment ArtworkSidebarBidAction_artwork on Artwork {
+      bidderStatus(live: true) {
+        active_bid {
+          __id
+        }
+      }
       sale {
+        registrationStatus {
```

Comment:
Really much cleaner to have it under artwork. I was wondering how would we deal with all this user/sale/artwork ids passing around. Thanks for taking it!